webmention.gemspec

```diff
@@ -11,24 +11,33 @@ Gem::Specification.new do |spec|
   spec.authors       = ['Aaron Parecki', 'Nat Welch']
   spec.email         = ['aaron@parecki.com']
 
-  spec.summary       = 'Webmention and pingback notification client'
-  spec.description   = 'A Ruby gem for sending webmention and pingback notifications.'
+  spec.summary       = 'Webmention notification client'
+  spec.description   = 'A Ruby gem for sending webmention notifications.'
   spec.homepage      = 'https://github.com/indieweb/webmention-client-ruby'
   spec.license       = 'Apache-2.0'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(bin|test)/}) }
 
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 2.0'
+  spec.metadata = {
+    'bug_tracker_uri' => "#{spec.homepage}/issues",
+    'changelog_uri'   => "#{spec.homepage}/blob/v#{spec.version}/CHANGELOG.md"
+  }
```

Comment:
Adding these metadata keys that flag URLs up to [the gem's page on RubyGems.org](https://rubygems.org/gems/webmention).